chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertificationBodyController.java

```diff
@@ -139,14 +141,14 @@ private CertificationBody create(final CertificationBody acbInfo)
     produces = "application/json; charset=utf-8")
     public CertificationBody updateAcb(@RequestBody final CertificationBody acbInfo) throws InvalidArgumentsException,
             EntityRetrievalException, JsonProcessingException, EntityCreationException, UpdateCertifiedBodyException,
-            SchedulerException, ValidationException {
+            SchedulerException, ValidationException, IllegalArgumentException, MessagingException {
 
         return update(acbInfo);
     }
 
     private CertificationBody update(final CertificationBody updatedAcb) throws InvalidArgumentsException,
             EntityRetrievalException, JsonProcessingException, EntityCreationException, UpdateCertifiedBodyException,
-            SchedulerException, ValidationException {
+            SchedulerException, ValidationException, IllegalArgumentException, MessagingException {
```

Comment:
I can see the MessagingException come from the manager, which I'm not sure about, but what is changing with the ACB that needs an IllegalArgumentException?